<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importDialog</class>
 <widget class="QDialog" name="importDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Import</string>
      </attribute>
      <widget class="QSpinBox" name="startLineSpinBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>51</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="filenameEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>251</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="openFileButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>25</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>251</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Seperator</string>
       </property>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Space (&quot; &quot;)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Komma (&quot;,&quot;)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>82</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Tab</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>irgendwas</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>other</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="seperatorEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>31</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>214</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Begin Import from Line</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>221</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>File Info</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Created:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Modified:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Owner:</string>
        </property>
       </widget>
       <widget class="QLabel" name="fileSizeLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fileCreatedLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fileModifiedLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fileOwnerLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>491</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Data Mapping</string>
      </attribute>
      <widget class="QComboBox" name="dateTimeComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>24</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="xComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>54</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="yComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>84</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="valueComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>114</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date/Time</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Axis</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Axis</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
      <widget class="QComboBox" name="dateEditComboBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>26</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Unix Time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yyyy/MM/dd hh:mm:ss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yyyy.MM.dd hh:mm:ss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dd.MM.yyyy hh:mm:ss</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="refComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>144</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>148</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ref Sw. State</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>importDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
